packages/sanity/src/desk/components/pane/PaneHeader.styles.tsx

```diff
@@ -1,21 +1,42 @@
 import {Box, Flex, Layer, rgba, TextSkeleton, Text, Theme, Card} from '@sanity/ui'
 import styled, {css} from 'styled-components'
 
-export const Root = styled(Layer)`
-  line-height: 0;
-  position: sticky;
-  top: 0;
-
-  &:not([data-collapsed]):after {
-    content: '';
-    display: block;
-    position: absolute;
-    left: 0;
-    right: 0;
-    bottom: -1px;
-    border-bottom: 1px solid var(--card-shadow-outline-color);
-  }
-`
+/**
+ * Left/right border offsets for header components (in px).
+ */
+const BORDER_OFFSET_X = 12
+
+interface RootProps {
+  $borderBottom?: boolean
+  'data-collapsed'?: string
+  'data-testid'?: string
+  $isContentScrollable?: boolean
+  $hasScrolledFromTop?: boolean
```

Comment:
These can safely be omitted, as `$borderBottom` is already derived from `isScrollable` and `hasScrolledFromTop` values, so we're probably sending more than we need.

Non JS identifiers (like `data-*` attributes) don't need to be manually typed, so these can also be omitted.